curriculum/challenges/english/25-front-end-development/workshop-library-manager/67116c175a77cb64c5c10d49.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 67116c175a77cb64c5c10d49
+title: Step 1
+challengeType: 1
+dashedName: step-1
+---
+
+# --description--
+
+In this project, you will learn higher-order array methods by building a library manager. The methods you'll learn include `.map()`, `.filter()`, `.reduce()` and `.forEach()`.
```

Comment:
Remember they learned about higher order functions in the previous lecture block. So the text will need to reflect that.

```suggestion
In the previous lecture videos, you learned how to work with higher order functions like `map`, `filter` and `reduce`.

In this workshop, you will build a library manager app that will give you an opportunity to practice working with different higher order functions.
```